planning/autoware_planning_validator/src/utils.cpp

```diff
@@ -155,36 +155,74 @@ std::pair<double, size_t> calcMaxCurvature(const Trajectory & trajectory)
   return {*max_curvature_it, index};
 }
 
+void calc_interval_distance(
+  const Trajectory & trajectory, std::vector<double> & interval_distance_arr)
+{
+  if (trajectory.points.size() < 2) {
+    interval_distance_arr = std::vector<double>(trajectory.points.size() - 1, 0.0);
+    return;
+  }
+
+  interval_distance_arr = std::vector<double>(trajectory.points.size() - 1, 0.0);
+  for (size_t i = 0; i < trajectory.points.size() - 1; ++i) {
+    const auto d = calc_distance2d(trajectory.points.at(i), trajectory.points.at(i + 1));
+    interval_distance_arr.at(i) = d;
+  }
```

Comment:
same to 

```suggestion
interval_distance_arr = std::vector<double>(trajectory.points.size() - 1, 0.0);

  if (trajectory.points.size() < 2) {
    return;
  }

  for (size_t i = 0; i < trajectory.points.size() - 1; ++i) {
    const auto d = calc_distance2d(trajectory.points.at(i), trajectory.points.at(i + 1));
    interval_distance_arr.at(i) = d;
  }
```

?

Reply:
addressed in https://github.com/autowarefoundation/autoware_universe/pull/10385/commits/3dafa16e15b9c1955c5c78914ca9722bf80411eb